### PR TITLE
Fix unstaking issues

### PR DESCRIPTION
**Pull request by jmoreira-valory**
Fix unstaking issue

Addresses issue #94

### CODE REVIEW

File: scripts/staking.py [hunk 1]
```diff
@@ -84,13 +79,25 @@
             next_ts = get_next_checkpoint_ts(
                 ledger_api, args.staking_contract_address
             )
-            if next_ts > time.time() and not args.skip_livenesss_check:
+
+            liveness_period = 24*60*60 # TODO Read from contract
```
Comment (Adamantios) [inline scripts/staking.py:83]: Will the TODO be implemented in another PR?

**Review by Adamantios — APPROVED** [review]

File: scripts/staking.py (hunk 1 above)
Comment (jmoreira-valory) [reply scripts/staking.py]: Fixed

**Review by Adamantios — APPROVED** [review]

**Review by 0xArdi — APPROVED** [review]
